modules/mono/glue/GodotSharp/GodotSharp/Core/Plane.cs

```diff
@@ -188,7 +162,7 @@ public bool HasPoint(Vector3 point, real_t epsilon = Mathf.Epsilon)
                 return null;
             }
 
-            return from + dir * -dist;
+            return from + (dir * -dist);
```

Comment:
Even better:

```suggestion
            return from - (dir * dist);
```

I see at least one other instance like this. I wonder if there's some way to automatically find unnecessary unary nots.